### DISCUSSION TITLE
Image Targets Compiler as a command lien tool

### DISCUSSION THREAD

**Discussion by GwenGuts** [hiukim/mind-ar-js#536] [Ideas]
Hi, 

First, I want to say that the image tracking feature is mind blowing ! It works very well with a less amount of jitters, it's incredible.

Next, your tool for create targets is working very well (https://hiukim.github.io/mind-ar-js-doc/tools/compile). My question was to know if it exists a command line tool for using it and automating the conversions of images, and, if not, is it something that you could implement ?

Thanks !

**Comment by jabbourdan** [comment]
Hey @GwenGuts,

Did you find the way to do it? or anyone can help?

**Comment by Nipanepiyush** [comment]
<img width="794" height="437" alt="colles" src="https://github.com/user-attachments/assets/4bfdf203-7b24-40f9-9400-b060c0ee6e36" />
